.ci/pitest.sh

```diff
@@ -152,7 +152,13 @@ pitest-javadoc)
 #   ;;
 
 *)
-  echo "Unexpected argument: $1"
+  if [[ -n "$1" && "--help" != "$1" ]]; then
+    echo "Unexpected argument: $1"
```

Comment:
Does something like $1 and beyond make sense here? I think it is like `$1-`

Reply:
`$1-` is a hack to turn an empty string into something comparable.  But we already tested this variable and know that it is not empty.

Reply:
I am just looking so that if the user provides multiple arguments, all are displayed as part of the `Unexpected argument` display.
Ex: 
> $.ci/pitest.sh bad1 bad2 bad3
> Unexpected argument: bad1 bad2 bad3

Reply:
I am ok with such error message.
This script should not be good for dummies, it imply understanding and ability to read script

Reply:
done. Now output is
```bash
$ .ci/pitest.sh foo bar baz
Unexpected argument: foo bar baz
Usage .ci/pitest.sh <profile>
To see the full list of supported profiles run '.ci/pitest.sh --list'
```